common/build.gradle.kts

```diff
@@ -101,11 +105,11 @@ buildkonfig {
             props.load(localPropsFile.inputStream())
         }
 
-        if (props.containsKey("mapbox.key")) {
-            buildConfigField(com.codingfeline.buildkonfig.compiler.FieldSpec.Type.STRING, "MAPBOX_KEY", "\"${props.getProperty("mapbox.key")}\"")
-        } else {
-            throw GradleException("mapbox.key not declared in local.properties")
-        }
+//        if (props.containsKey("mapbox.key")) {
+//            buildConfigField(com.codingfeline.buildkonfig.compiler.FieldSpec.Type.STRING, "MAPBOX_KEY", "\"${props.getProperty("mapbox.key")}\"")
+//        } else {
+//            throw GradleException("mapbox.key not declared in local.properties")
+//        }
```

Comment:
This will need to be uncommented.

You can add `MAPBOX_KEY=<mapbox key here>` to your `local.properties` file to fix the build warning.